.tekton/push.yaml

```diff
@@ -85,6 +85,30 @@ spec:
               done
         taskRef:
           name: update-infra-deployments
+      - name: update-ec-policies
+        runAfter:
+          - build-bundles
+        params:
+          - name: ORIGIN_REPO
+            value: $(params.git-url)
+          - name: REVISION
+            value: $(params.revision)
+          - name: TARGET_GH_REPO
+            value: hacbs-contract/ec-policies
+          - name: SCRIPT
+            value: |
+              export DIGEST='$(results.build-container.IMAGE_DIGEST)'
+              export BUNDLE='quay.io/redhat-appstudio/build-templates-bundle'
+              export TAG='$(params.revision)'
+              # 720h == 30 days
+              export EFFECTIVE_ON="$(yq --null-input 'now + "720h" | tz("UTC")')"
+
+              yq '."pipeline-bundles"[env(BUNDLE)] |=
```

Comment:
@simonbaird, since this is now using the `appstudio-utils` image, I could turn this into a call to ec-cli. Do you think it makes sense to introduce this logic to it?

I'm thinking we may want a more complex behavior when dealing with Task bundles since a repo has multiple "streams", e.g. `-1`, `-2`, `-3`, etc.

Something like this:

```
$ ec track bundles --file data/acceptable_tekton_bundles.yml \
    quay.io/redhat-appstudio/build-templates-bundle:ad4c34b71d468a991450b0091cd96765471ed82d \
    quay.io/redhat-appstudio/appstudio-tasks:ad4c34b71d468a991450b0091cd96765471ed82d-1 \
    quay.io/redhat-appstudio/appstudio-tasks:ad4c34b71d468a991450b0091cd96765471ed82d-2 \
    quay.io/redhat-appstudio/appstudio-tasks:ad4c34b71d468a991450b0091cd96765471ed82d-3 \
```

Reply:
Also, the digest of each bundle is not tracked anywhere in this push pipeline. ec-cli could resolve it.

Reply:
Done with latest update.